tests/filtering/test_derive_adduct_from_name.py

```diff
@@ -1,7 +1,8 @@
 import pytest
```

Comment:
The tests do not yet follow the same structure as the filter functions. For consistency this might be good to change as well.